Array_1_Problem_3.java

```diff
@@ -0,0 +1,135 @@
+//Time Complexity : O(n)
+//Space Complexity: O(n)
```

Comment:
Space complexity can be improved. Please see the lecture video if you missed the class and try the same problem iteratively and recursively